src/main/prepared_statement_data.cpp

```diff
@@ -1,6 +1,7 @@
 #include "duckdb/main/prepared_statement_data.hpp"
 #include "duckdb/execution/physical_operator.hpp"
 #include "duckdb/parser/sql_statement.hpp"
+#include "duckdb/execution/operator/projection/physical_projection.hpp"
```

Comment:
This include is no longer necessary, right?

Reply:
removed. thanks ;)